libs/perun/dialogs/src/lib/delete-entity-dialog/get-string-for-dialog-type.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'getStringForDialogType',
```

Comment:
I would say this name is both unnecessarily long and not very descriptive. Perhaps something along the lines of `actionType` could be used.